tests/PHPUnit/System/expected/test_SiteSearch_Actions.getSiteSearchKeywords_firstSite_lastN__API.getProcessedReport_day.xml

```diff
@@ -90,6 +90,6 @@
 	<reportTotal>
 		<nb_visits>8</nb_visits>
 		<nb_hits>13</nb_hits>
-		<exit_nb_visits>1</exit_nb_visits>
+		<exit_nb_visits>2</exit_nb_visits>
```

Comment:
Another example where one can see that it fixes report total calculation as mentioned before.